core/ir/aarch64/instr_create_api.h

```diff
@@ -4970,4 +4970,199 @@
 #define INSTR_CREATE_movprfx_vector(dc, Zd, Zn) \
     instr_create_1dst_1src(dc, OP_movprfx, Zd, Zn)
 
+/**
+ * Creates a SQADD instruction.
+ *
+ * This macro is used to encode the forms:
+ * \verbatim
+ *    SQADD   <Zdn>.<Ts>, <Zdn>.<Ts>, #<imm>, <shift>
+ * \endverbatim
+ * \param dc   The void * dcontext used to allocate memory for the #instr_t.
```

Comment:
Nit: do the start of description lines need to be aligned to each other to look nice and tidy? Or does the HTML renderer take care of that?

Reply:
I believe the renderer will split on whitespace, so should be fine